### PR TITLE
added footer

### PR DESCRIPTION
**Pull request by AbhipsaGuru1012**
## Related Issuse or bug
  - Info about Issue or bug

Fixes: #26 
## Type of change
Added footer

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |

Updated Screenshot:
hackstation

### CODE REVIEW

**Comment by ashishnagpal2498** [comment]
Hi, I just checked the code.... You weren't suppose to add bootstrap. As I have already mentioned only CSS. So could you please change to CSS flex boxes ?

**Comment by AbhipsaGuru1012** [comment]
> Hi, I just checked the code.... You weren't suppose to add bootstrap. As I have already mentioned only CSS. So could you please change to CSS flex boxes ?

Okay, I will make the required changes

**Comment by AbhipsaGuru1012** [comment]
@ashishnagpal2498 I have made the required changes. Please review.

File: frontend/src/index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+// import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (ashishnagpal2498) [inline frontend/src/index.js:6]: Please remove this line.

File: frontend/src/components/layout/Footer.js [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
-			<div className="footer-logo">
-				<img src="" alt="footer"/>
-			</div>
-			<div className="footer-options">
-				
-			</div>
+				<div className="footer-options">
+					<div className="footer-logo">
+						<a href="/">
+							<img src="https://user-images.githubusercontent.com/36301481/104619223-a0b0ad00-56b3-11eb-84ce-9b845b1729b5.png" alt="footer"/>
```
Comment (ashishnagpal2498) [inline frontend/src/components/layout/Footer.js:11]: Download the image to local and add in public folder.

File: frontend/package.json [hunk 3]
```diff
@@ -9,8 +9,10 @@
     "axios": "^0.20.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-icons": "^4.1.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.3"
+    "react-scripts": "3.4.3",
```
Comment (ashishnagpal2498) [inline frontend/package.json:14]: why need of this package ?

File: frontend/package.json [hunk 4]
```diff
@@ -9,8 +9,10 @@
...
+    "styled-components": "^5.2.1"
```
Comment (ashishnagpal2498) [inline frontend/package.json:15]: and this?

**Review by ashishnagpal2498 — CHANGES_REQUESTED** [review]

**Comment by ashishnagpal2498** [comment]
Please check for CSS of button. Looks bit odd. Also the page is exceeding the viewport in x- axis. Check that. 
image

File: frontend/package.json (hunk 3 above)
Comment (AbhipsaGuru1012) [reply frontend/package.json]: @ashishnagpal2498 react-scripts is in the original package.json file. I haven't added that.

**Comment by AbhipsaGuru1012** [comment]
@ashishnagpal2498 I have made the changes. Please review this version.

**Comment by ashishnagpal2498** [comment]
Not responsive ? 
image

**Comment by AbhipsaGuru1012** [comment]
> Not responsive ?
> image

@ashishnagpal2498 now it's responsive

File: frontend/src/components/layout/Footer.js [hunk 5]
```diff
@@ -1,15 +1,47 @@
...
 const Footer = () => {
 	return (
 		<div className="footer-wrapper">
-			<div className="footer-logo">
-				<img src="" alt="footer"/>
-			</div>
-			<div className="footer-options">
-				
-			</div>
+				<div className="footer-options">
```
Comment (ashishnagpal2498) [inline frontend/src/components/layout/Footer.js:8]: No need for this div.

File: frontend/src/components/styles/Footer.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    bottom: 0;
+}
+.description{
+    justify-content: center;
+}
+h4{
+    font-size: 1.5rem;
+}
+.footer-logo, .about, .contact{
+    width: 30%;
```
Comment (ashishnagpal2498) [inline frontend/src/components/styles/Footer.css:20]: If you are using flex, you don't need width.

File: frontend/src/components/styles/Footer.css [hunk 7]
```diff
@@ -0,0 +1,63 @@
+img{
+    height: 200px;
+    width: 200px;
+}
+.footer-wrapper{
+    display: inline-block;
```
Comment (ashishnagpal2498) [inline frontend/src/components/styles/Footer.css:6]: Why it should be inline-block ?

**Review by ashishnagpal2498 — CHANGES_REQUESTED** [review]
You are using too much of redundant properties. Please use read and use flex boxes well. Try once. If you don't get I'll help you. Just let me know.
See, it's looking bit odd
image

Unused properties.
image

**Comment by AbhipsaGuru1012** [comment]
@ashishnagpal2498 okay, I will make the changes. I am facing certain issues while making it responsive. I will try to complete it and get back soon

**Comment by ashishnagpal2498** [comment]
> 
> 
> @ashishnagpal2498 okay, I will make the changes. I am facing certain issues while making it responsive. I will try to complete it and get back soon

Its already been delayed too much. Please provide a PR by tomorrow.

**Pull request closed without merging**